src/macfc-security-hub-sync.ts

```diff
@@ -428,6 +631,7 @@ export class SecurityHubJiraSync {
     }
     let newIssueInfo
     try {
+      console.log(newIssueData)
```

Comment:
Is this console.log necessary when it is not in debug mode?

Reply:
removed